tox.ini

```diff
@@ -238,6 +263,46 @@ commands =
     coverage report --ignore-errors --skip-covered --omit "wandb/vendor/*"
     codecov -e TOXENV -F unittest
 
+# temporary until we can consolidate coverage collection paths
+[testenv:unit-covercircle]
+skip_install = true
+basepython = python3
+passenv = CI CIRCLECI CIRCLE_* CODECOV_* TOXENV
+deps =
+    pytest
+    coverage
+    codecov
+setenv =
+    CIRCLE_BUILD_NUM={env:CIRCLE_WORKFLOW_ID}
+whitelist_externals =
+    mkdir
+    bash
+    cp
+commands =
+    mkdir -p cover-results
+    /usr/bin/env bash -c '{envpython} -m coverage combine {toxworkdir}/unit-s*-py*/.coverage*'
+    /usr/bin/env bash -c '{envpython} -m coverage xml --ignore-errors'
+    cp .coverage coverage.xml cover-results/
+    coverage report --ignore-errors --skip-covered --include "wandb/*" --omit "wandb/vendor/*"
```

Comment:
Why use `--ignore-errors`?

Reply:
Im not sure when this was added (copied from other similar tox coverage environments),  this line is just for the text output, i think there were some cases where files were slightly corrupted and we got no output, it might not be an issue anymore.